internal/kafka/internal/services/clusters.go

```diff
@@ -783,6 +787,7 @@ func (c *clusterService) FindStreamingUnitCountByClusterAndInstanceType() (Kafka
 	if err := dbConn.Model(&dbapi.KafkaRequest{}).
 		Select("cloud_provider, region, count(1) as Count, size_id, cluster_id, instance_type").
 		Group("size_id, cluster_id, cloud_provider, region, instance_type").
+		Where("status not in (?)", kafkaStatusesThatNoLongerConsumesResourcesInTheDataPlane).
```

Comment:
Can we update the method documentation to describe this?

Reply:
Good catch! I always forget about this. I'm on it.

Reply:
I've done the update in the interface

Reply:
:+1: